### DISCUSSION TITLE
drawings are not visible with pycharm

### DISCUSSION THREAD

**Discussion by mohammednooraldeen** [rdkit/rdkit#5867] [Q&A]
I cannot make drawings visible from rdkit or py3dmol in pycharm unless being images.
when I use IPythonConsole, even drawings are not visible and when use Draw.MolsToGridImage(mols).show() it gives error 

rendering figures using py3dmol is also not giving drawings.
for example:
p_crippen=py3Dmol.view(width=600, height=400)
p_crippen.addModel(Chem.MolToMolBlock(ref_mol1),'sdf')
p_crippen.render()
p_crippen  # ==> shows nothing
p_crippen.show() # ==> shows nothing

**Comment by greglandrum** [comment]
Given that I don't use pycharm, I'm afraid that I can't help here at all.
Maybe someone else will be able to help out.

**Comment by mohammednooraldeen** [comment]
I found a way. I installed Juptyer within Pycharm and do my work as jubtyer notebook file instead of python file. so things go OK
